website/docs/docs/build/unit-tests.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Unit tests"
+sidebar_label: "Unit tests"
+description: "Learn how to use unit tests on your SQL models."
```

Comment:
![Screenshot 2024-02-14 at 11 02 21 AM](https://github.com/dbt-labs/docs.getdbt.com/assets/53586774/187f9092-825f-4ae2-a4a0-9280e02aa573)

I can see this on version 1.7 - is that expected?

Reply:
@graciegoheen  Yes, we made this call because versionless Cloud users aren't necessarily going to know to choose the 1.8 documentation to view this feature, so we made it visible for all. If you'd rather this whole page be versioned, we can do that, but I think views and discoverability will take a hit

Reply:
Nope that makes sense, just wanted to confirm it was intentional!